### PR TITLE
Added output variable with the value of the newly created tag

### PR DESCRIPTION
**Pull request by falko189**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -97,6 +97,7 @@ Toolkit.run(async tools => {
     newVersion = execSync(`npm version --git-tag-version=false ${version}`).toString().trim()
     newVersion = `${process.env['INPUT_TAG-PREFIX']}${newVersion}`
     console.log('new version:', newVersion)
+    console.log(`::set-output name=newTag::${newVersion}`)
```
Comment (phips28) [inline index.js:100]: @falko189 can you point to the official documentation of this?

Comment (falko189) [reply index.js:100]: https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#outputs

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter

I think a console log is enough? or should it be inside the docker file as actual echo

Comment (falko189) [reply index.js:100]: I also realised that in case we merge this I should update the doc to reflect the change.

Comment (phips28) [reply index.js:100]: I am not sure, but I hope you tested it?!

Comment (falko189) [reply index.js:100]: Not sure how to test this on public repos, I have worked with actions only on private repos 🤔

Comment (phips28) [reply index.js:100]: doesnt matter if public or private repo. the action is public and can be used. so you should use your branch as action and test your behavior if you get the output you want.

Comment (falko189) [reply index.js:100]: Closing this to perform testing first :D

**Pull request closed without merging**